src/main/java/net/minecraftforge/common/ForgeRecipeBookCategories.java

```diff
@@ -0,0 +1,54 @@
+package net.minecraftforge.common;
+
+import com.google.common.collect.ImmutableList;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraftforge.common.extensions.IForgeRecipeBookCategory;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+public class ForgeRecipeBookCategories<T extends IRecipeType<?>> extends net.minecraftforge.registries.ForgeRegistryEntry<IForgeRecipeBookCategory<?>> implements IForgeRecipeBookCategory<T> {
```

Comment:
Should this not be named: `ForgeRecipeBookCategory` ? Since this is one category and not multiple ones?

Reply:
Right, I will change it